spec/dynamoid/dumping_spec.rb

```diff
@@ -174,19 +174,6 @@
         expect(raw_attributes(obj)[:sent_at]).to eql(BigDecimal('1532131200.0'))
       end
 
-      it 'does not loose precision and can be used as sort key', :bugfix do
```

Comment:
Could we keep this spec in another form?  Do we already have specs testing the precision retention of the alternative methods?

Reply:
@pboling Could you please clarify what form you mean?

Reply:
Oh I think I misunderstood.  So there is no current way to not lose precision, but that will be added back in a future PR. 👍

Reply:
@pboling 

Yep. The issue is reproduced only with JRuby constantly and isn't reproduced with MRI...

But there is no any way to ensure that time related calculations don't lead to losing precision even on MRI unless core team declares this.

But they say only:
> The lowest digits of to_f and nsec are different because IEEE 754 double is not accurate enough to represent the exact number of nanoseconds since the Epoch.
> The more accurate value is returned by nsec.

https://ruby-doc.org/core-2.5.0/Time.html#nsec-method